packages/web-components/@sfx/products/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@sfx/products",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/index.ts",
+  "types": "dist/index.d.ts",
```

Comment:
This is missing a `"modules": "src/index.ts"` declaration. Add it immediately above.